Taskfile.yml

```diff
@@ -245,6 +240,14 @@ tasks:
           DATA_DIR: "{{.OUTPUT_DIR}}"
     cmds:
       - "rm -rf '{{.OUTPUT_DIR}}'"
+      - "mkdir -p '{{.OUTPUT_DIR}}'"
+      - |-
+        cd "server"
+        rsync -a \
+        package.json package-lock.json \
```

Comment:
stupid question, don't we need to sync "src" as well?

Reply:
not at all. How TypeScript compilation works is that the compiler performs type-checking then transpiles the TypeScript code into JavaScript code. 

After we run `npm run build` to perform the compilation, the JS code would have been the `dist` folder and the sources stored in `src` can be thrown away.